docs: Add docs about configuring browser for HTTPS certificates (#7075)
ddev/ddev: docs/content/users/install/ddev-installation.md

    mkcert -install
    ```

>     Optionally, [configure your browser](configuring-browsers.md).

Reviewer: ```suggestion
    For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```

    curl -fsSL https://ddev.com/install.sh | bash
    ```

>     Optionally, [configure your browser](configuring-browsers.md).

Reviewer: ```suggestion
For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```

    mkcert -install
    ```

>     Optionally, [configure your browser](configuring-browsers.md).

Reviewer: ```suggestion
For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```

    mkcert -install
    ```

>     Optionally, [configure your browser](configuring-browsers.md).

Reviewer: ```suggestion
For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```

    mkcert -install
    ```

>     Optionally, [configure your browser](configuring-browsers.md).

Reviewer: ```suggestion
For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```

    mkcert -install
    ```

>     Optionally, [configure your browser](configuring-browsers.md).

Reviewer: ```suggestion
For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```

    curl -fsSL https://ddev.com/install.sh | bash
    ```

>     Optionally, [configure your browser](configuring-browsers.md).

Reviewer: ```suggestion
For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```

        iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/ddev/ddev/main/scripts/install_ddev_wsl2_docker_desktop.ps1'))
        ```

>     10. Optionally, [configure your browser](configuring-browsers.md).

Reviewer: ```suggestion
    10. For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```

    13. In WSL2, run `mkcert -install`.

>     14. Optionally, [configure your browser](configuring-browsers.md).

Reviewer: ```suggestion
    14. For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```

    * As a one-time initialization, run `mkcert -install`, which may require your `sudo` password.

        If you don’t have `mkcert` installed, download the [latest release](https://github.com/FiloSottile/mkcert/releases) for your architecture and `sudo mv <downloaded_file> /usr/local/bin/mkcert && sudo chmod +x /usr/local/bin/mkcert`.

>     * Optionally, [configure your browser](configuring-browsers.md).

Reviewer 2: ```suggestion
    * For unusual browsers and situations that don't automatically support the `mkcert` certificate authority, [configure your browser](configuring-browsers.md).
```
